File: csharp/MapleStory.OpenAPI/Src/Dto/UnionRaiderBlockPositionDTO.cs
using Newtonsoft.Json;

namespace MapleStory.OpenAPI.Dto
{
    /// <summary>
    /// 유니온 블록이 차지하고 있는 영역 좌표들
    /// </summary>
    public class UnionRaiderBlockPositionDTO
    {
        /// <summary>
        /// 블록 X좌표
        /// </summary>
        [JsonProperty("x")]
        public long X { get; set; }

        /// <summary>
        /// 블록 Y좌표
        /// </summary>
        [JsonProperty("y")]
        public long Y { get; set; }
    }
}

File: java/src/main/java/dev/spiralmoon/maplestory/api/tms/dto/union/UnionRaiderBlockPositionDTO.java
package dev.spiralmoon.maplestory.api.tms.dto.union;

import com.fasterxml.jackson.annotation.JsonProperty;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;
import lombok.ToString;

/**
 * 方塊佔領區域的座標
 */
@AllArgsConstructor
@NoArgsConstructor
@Data
@ToString
public class UnionRaiderBlockPositionDTO implements dev.spiralmoon.maplestory.api.common.dto.union.UnionRaiderBlockPositionDTO {

    /**
     * 方塊 x 座標
     */
    @JsonProperty("x")
    private long x;

    /**
     * 方塊 y 座標
     */
    @JsonProperty("y")
    private long y;
}
